sys-account/server/authn/pkg/utilities/helpers.go

```diff
@@ -1,11 +1,10 @@
 package utilities
```

Comment:
These DB utilities should eventually be part of the sys-core util package.
once the DB code settles down we can do that.
All Modules can then use them too.

Reply:
Thank you, i'll wait until DB layer settles down, and will revisit this again.